docker/onadata-uwsgi/docker-compose.yml

```diff
@@ -14,13 +14,7 @@ services:
   queue:
     image: rabbitmq
   web:
-    build:
-      context: .
-      dockerfile: Dockerfile
-    image: onadata:latest
-    volumes:
-      # For local development
-      - ../../.:/srv/onadata
+    image: onaio/onadata:latest
```

Comment:
The point about doing local builds was for development purposes. In my last years of writing code on this repositories this has been the primary way my dev environment worked, from within the container. Perhaps add a docker-compose.prod.yml file instead?